README.md

```diff
@@ -27,35 +31,48 @@ cd ../../../
 rm -rf cpprestsdk
 ```
 
-### Windows
-Windows dependencies are managed through Vcpkg and Nuget.
-
-```
-vcpkg install cpprestsdk:x64-windows
-vcpkg install zlib:x64-windows
-vcpkg install boost-system:x64-windows
-vcpkg install boost-program-options:x64-windows
-vcpkg install boost-test:x64-windows
-vcpkg install boost-uuid:x64-windows
-```
-
-## Build
+### Build
 
 ```
 mkdir build
 cd build
 cmake ..
 make
+```
 
-# Test
+### Test
+```
 mkdir build
 cd build
 cmake .. -DTURN_OFF_DEVIRTUALIZE=On
 make rl_clientlib_test
 make test
 ```
 
+## Windows
+Windows dependencies are managed through Vcpkg and Nuget.
+
+```
+vcpkg install cpprestsdk:x64-windows
+```
+
+### Build + Test
+
+Open `rl.sln` in Visual Studio 2017.
+
 ### Experimental - CMake on Windows
+Using CMake is an alternate way to configure and build the project. Currently it only supports the C++ projects.
+All dependencies are managed through Vcpkg:
+```
+vcpkg install cpprestsdk:x64-windows
+vcpkg install zlib:x64-windows
+vcpkg install boost-system:x64-windows
+vcpkg install boost-program-options:x64-windows
+vcpkg install boost-test:x64-windows
+vcpkg install boost-uuid:x64-windows
+```
```

Comment:
This instruction is not up to date, missing flatbuffers. It should align with build-windows.cmd

Reply:
vcpkg supports response files, so it might be easier to put the packages into a file (packages.txt) and install everything as:
```
>vcpkg install @packages.txt
```

Reply:
@ras0219-msft That's a handy new feature, thanks for letting us know! I am hesitant to do that right now since the default on Windows is x86, we would then need a specific package requirements file for each platform as we target x64 by default